content/docs/user-guide/packaging/project-export-ios.md

```diff
@@ -0,0 +1,84 @@
+---
+linkTitle: Project Export iOS
+title: Project Export for iOS
+description: (Experimental) Learn how to use the Project Export CLI to create an Xcode project to build and deploy for iOS.
+toc: true
+weight: 450
+---
+In this article, we will go over the process to use project export tooling as part of the process to building and deploying games to the iOS platform. First, we will use the iOS export script to generate the necessary Xcode project file and prepare assets. Afterwards we will walk through how to ensure Xcode is setup properly to build an app for iOS.
+
+For this demonstration, we will use the O3DE Atom Sample Viewer project as our reference point for ensuring proper behavior. Results may vary for other projects.
+
+{{< note >}}
+To learn more about the project export tooling, please consult the page [Project Export CLI Tool.](project-export-cli)
+{{< /note >}}
+{{< important >}}
+The iOS export functionality is only available on macOS, which is currently experimental for O3DE.
+{{< /important >}}
+
+## Prerequisites
+First, make sure to follow the pre-requisites of the [Project Export CLI Tool page.](project-export-cli) Afterwards, ensure that the [O3DE Atom Sample Viewer project](https://github.com/o3de/o3de-atom-sampleviewer) is cloned onto your machine, and is properly registered with your copy of O3DE. You can register the project using:
+```
+<O3DE_ENGINE>\scripts\o3de register -pp C:\path\to\o3de-atom-sampleviewer
```

Comment:
Please switch all the slashes to forward and drop the drive letter prefixes
```suggestion
<O3DE_ENGINE>/scripts/o3de register -pp /path/to/o3de-atom-sampleviewer
```

Reply:
Also note that on Mac, as with Linux, you must include the extension `.sh`. Windows will search for .bat/.cmd/.exe if it is omitted.